src/Symfony/Bundle/FrameworkBundle/Translation/Translator.php

```diff
@@ -102,7 +108,8 @@ public function warmUp($cacheDir)
         }
 
         $locales = array_merge($this->getFallbackLocales(), [$this->getLocale()], $this->resourceLocales);
-        foreach (array_unique($locales) as $locale) {
+        $localesToWarmUp = $this->enabledLocales ?? $locales;
```

Comment:
you're deleting this `array_unique($locales)` function, is it not needed anymore?

Reply:
That was a mistake. Fixed. Thanks.